src/styles/blocks/calender.scss

```diff
@@ -0,0 +1,56 @@
+.calender {
+  width: $width-calc;
+  display: flex;
+  flex-direction: row;
+
+  flex-wrap: wrap;
+  gap: $box-gap;
+  padding: $padding;
+
+  &__box {
+    background-color: $calender-color;
+    height: $box-size;
+    line-height: $box-size;
+    display: flex;
+
+    justify-content: center;
+    align-items: center;
+    color: black;
+    font-family: Arial, Helvetica, sans-serif;
+
+    font-size: 30px;
+    text-align: center;
```

Comment:
Same here you already used justify-content therefore no need of text-align. You may consider removing this.